src/main/java/core/basesyntax/operationhandler/BalanceHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.operationhandler;
+
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.Storage;
+
+public class BalanceHandler implements OperationHandler {
+    @Override
+    public void handleOperation(FruitTransaction transaction, Storage storage) {
+        System.out.println("Current balance:");
+        for (String fruit : Storage.fruits.keySet()) {
+            System.out.println(fruit + ": " + Storage.fruits.get(fruit));
+        }
+    }
```

Comment:
This method should change state of storage but it does nothing except priniting some info into console.

Balance is revision operation, it should replace current amount of fruit to new one